bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/config/BpdmSecurityConfigurerAdapterImpl.kt

```diff
@@ -38,22 +38,25 @@ class BpdmSecurityConfigurerAdapterImpl(
     }
 
     override fun configure(http: HttpSecurity) {
-        http.csrf().disable()
-            .cors()
-            .and().sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-            .and().authorizeHttpRequests()
-            .requestMatchers(AntPathRequestMatcher(ALL_API_PATHS, HttpMethod.OPTIONS.name())).permitAll()
-            .requestMatchers(AntPathRequestMatcher("/")).permitAll() // forwards to swagger
-            .requestMatchers(AntPathRequestMatcher("/docs/api-docs/**")).permitAll()
-            .requestMatchers(AntPathRequestMatcher("/ui/swagger-ui/**")).permitAll()
-            .requestMatchers(AntPathRequestMatcher("/actuator/health/**")).permitAll()
-            .requestMatchers(AntPathRequestMatcher(ALL_API_PATHS, HttpMethod.GET.name())).authenticated()
-            .requestMatchers(AntPathRequestMatcher("/api/catena/**/search", HttpMethod.POST.name())).authenticated()
-            .requestMatchers(AntPathRequestMatcher("/api/catena/**/filter", HttpMethod.POST.name())).authenticated()
-            .requestMatchers(AntPathRequestMatcher("/error")).permitAll()
-            .requestMatchers(AntPathRequestMatcher(ALL_API_PATHS)).hasRole("add_company_data")
-            .and().oauth2ResourceServer()
-            .jwt()
-            .jwtAuthenticationConverter(CustomJwtAuthenticationConverter(securityConfigProperties.clientId))
+        http.csrf { it.disable() }
+        http.cors {}
+        http.sessionManagement { it.sessionCreationPolicy(SessionCreationPolicy.STATELESS) }
+        http.authorizeHttpRequests {
+            it.requestMatchers(AntPathRequestMatcher(ALL_API_PATHS, HttpMethod.OPTIONS.name())).permitAll()
+            it.requestMatchers(AntPathRequestMatcher("/")).permitAll() // forwards to swagger
+            it.requestMatchers(AntPathRequestMatcher("/docs/api-docs/**")).permitAll()
+            it.requestMatchers(AntPathRequestMatcher("/ui/swagger-ui/**")).permitAll()
+            it.requestMatchers(AntPathRequestMatcher("/actuator/health/**")).permitAll()
+            it.requestMatchers(AntPathRequestMatcher(ALL_API_PATHS, HttpMethod.GET.name())).authenticated()
+            it.requestMatchers(AntPathRequestMatcher("/api/catena/**/search", HttpMethod.POST.name())).authenticated()
+            it.requestMatchers(AntPathRequestMatcher("/api/catena/**/filter", HttpMethod.POST.name())).authenticated()
+            it.requestMatchers(AntPathRequestMatcher("/error")).permitAll()
+            it.requestMatchers(AntPathRequestMatcher(ALL_API_PATHS)).hasRole("add_company_data")
```

Comment:
I think in the scope of this pull request we can also align the Pool's authorization behaviour. I would expect the same as the Gate here, since we now configure authorization of each endpoint individually over the PreAuthorize annotation. Everything that is not covered by a PreAuthorize annotation would then be at least authenticated.

Reply:
Done.
Also I changed AntPathRequestMatcher to the antMatcher() shortcut.